src/sql/src/session/vars/definitions.rs

```diff
@@ -669,6 +669,12 @@ pub static PG_TIMESTAMP_ORACLE_CONNECTION_POOL_TTL_STAGGER: VarDefinition = VarD
     false,
 );
 
+pub static UNSAFE_IS_SUPERUSER: VarDefinition = VarDefinition::new(
+    "unsafe_is_superuser",
+    value!(bool; false),
+    "If true, the user will be treated as a superuser for RBAC checks.",
+    true,
+);
```

Comment:
I think it's worth adding a test to this file, https://github.com/MaterializeInc/materialize/blob/main/src/environmentd/tests/server.rs, that starts a server without unsafe mode and asserting that we do not allow setting this variable.

Reply:
Probably also worth testing that setting this variable in unsafe mode actually gives the user superuser privileges.